### ISSUE TITLE
What is the intended behavior of the `CopyingCollector` policy?

### ISSUE THREAD

**Issue by aarontabor** [GarCoSim/TraceFileSim#12]
After a closer investigation of the `CopyingCollector` policy, I am a little confused as to its intended behavior. It appears to be attempting to perform split-heap (i.e. "new-space"/"old-space") copying, where, during each collection, the surviving objects are copied to the other half of the heap. This is exactly the behavior that is implemented in the `TraversalCollector`.

Does anyone know what the original motivation (and the distinction in intended behavior) behind the `CopyingCollector` and the `TraversalCollector` was?

**Comment by konsna** [comment]
They are basically both copying collectors. One of them was implemented by me and only worked with the simulated allocator. The other one implemented by Marcel in his simulator and used an early version of the real allocator. It was later moved to this simulator.

**Comment by aarontabor** [comment]
Would it be reasonable to remove one of them from the project?

**Comment by konsna** [comment]
Yes. Once one of them is working properly (It looks like Traversal is the one most up to date), we can remove the other one.
